gpsr/dependencies.go

```diff
@@ -116,3 +117,19 @@ func (ip InstallPlan) InvertDependencies() map[string][]string {
 	}
 	return idb
 }
+
+func (ip *InstallPlan) Pack(pkgNexus *cran.PkgNexus) {
```

Comment:
If I go through with this, I'll be sure to pick a better name than "Pack." The idea is that I'm packing the installPlan with its own PkgDl objects so that the install command doesn't have to recalculate these.